### PR TITLE
Update stay-on-top-of-updates.md

### PR DESCRIPTION
**Pull request by msbemba**
Add a note that only Global administrators can make changes to Release preferences.

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Office365-Admin/manage/stay-on-top-of-updates.md [hunk 1]
```diff
@@ -36,4 +36,6 @@ With Office 365, you receive new product updates and features as they become ava
...
 |**Roadmap** <br/> |Visit the Microsoft 365 Roadmap to learn about features that have been launched, are rolling out, are in development, have been cancelled, or previously released. The roadmap is the official site for Office 365 updates and changes.  <br/> |Visit the [Microsoft 365 Roadmap](https://www.microsoft.com/microsoft-365/roadmap) frequently and learn about planned updates and releases.  <br/> |
 |**Blogs** and **Community** <br/> |Visit Office Blogs, Microsoft Community, and Microsoft Tech Community to learn more details about changes in Office 365 and share experiences with other users.  <br/> |Visit [Office Blogs](https://blogs.office.com/). Visit [Microsoft Community](http://answers.microsoft.com). Visit [Microsoft Tech Community](https://techcommunity.microsoft.com).  <br/> |
-   
+
+> [!NOTE]
+> You need to be a Global Administrator to make changes to Release preferences.
```
Comment (JohanFreelancer9) [inline Office365-Admin/manage/stay-on-top-of-updates.md]:
```suggestion
> You need to be a global administrator to make changes to Release preferences.
```

File: Office365-Admin/manage/stay-on-top-of-updates.md [hunk 2]
```diff
@@ -36,4 +36,6 @@ With Office 365, you receive new product updates and features as they become ava
...
 |**Roadmap** <br/> |Visit the Microsoft 365 Roadmap to learn about features that have been launched, are rolling out, are in development, have been cancelled, or previously released. The roadmap is the official site for Office 365 updates and changes.  <br/> |Visit the [Microsoft 365 Roadmap](https://www.microsoft.com/microsoft-365/roadmap) frequently and learn about planned updates and releases.  <br/> |
 |**Blogs** and **Community** <br/> |Visit Office Blogs, Microsoft Community, and Microsoft Tech Community to learn more details about changes in Office 365 and share experiences with other users.  <br/> |Visit [Office Blogs](https://blogs.office.com/). Visit [Microsoft Community](http://answers.microsoft.com). Visit [Microsoft Tech Community](https://techcommunity.microsoft.com).  <br/> |
-   
+
+> [!NOTE]
+> You need to be a global administrator to make changes to Release preferences.
```
Comment (mypil) [inline Office365-Admin/manage/stay-on-top-of-updates.md]:
```suggestion
> You need to be a global administrator to make changes to release preferences.
```

**Comment by mypil** [comment]
@cmcatee-MSFT - Recommended changes has been applied. Kindly review for merging. Thank you.

**Pull request merged**